packages/ui/src/stubs/expression-definition-schema.ts

```diff
@@ -1643,3 +1643,451 @@ export const expressionDefinitions: KaotoSchemaDefinition['schema']['definitions
     required: ['expression'],
   },
 };
+
```

Comment:
Feel free to remove the properties that are not in the definition objects to make the schema smaller :smiley:

Reply:
I can remove some but not all. If the no. is lesser than 6, then `Typeahead` field wont be rendered.